booking.js

```diff
@@ -1,14 +1,17 @@
 const { v4: uuidv4 } = require('uuid');
 class Booking{
-        constructor(client, barber, date) {
-        
-            this.client = client       
-            this.barber = barber        
-            this.date = date
-            const bookingId = uuidv4();
-            this.bookingId = bookingId
+    constructor(client, barber, date) { 
+        this.client = client       
+        this.barber = barber        
+        this.date = date
+        const bookingId = uuidv4();
+        this.bookingId = bookingId
             //console.log(bookingId)
         }
+
+        addBooking(booking) {
```

Comment:
why do you have this `addBooking` function here?

Reply:
I removed the addBooking function and created a new booking function.